maintenance.md

```diff
@@ -668,14 +668,14 @@ The libsystemd version should be the same with the one running on [the stable Fl
 3. Update `REDIS_VERSION` in `Dockerfile`.
 4. Update `BRANCH` and `TAG` files.
 
-## registry
+## distribution
```

Comment:
It would be nice to move this section above because each sections seems sorted alphabetically.